jetstream/search-result-de-duplication.toml

```diff
@@ -0,0 +1,95 @@
+
+## Impressions
+[metrics.history_impressions]
+select_expression = "COUNT(DISTINCT IF(is_terminal AND result.product_result_type = 'history', event_id, NULL))"
+data_source = "urlbar_events_unnested_results"
+[metrics.history_impressions.statistics.bootstrap_mean]
+
+[metrics.bookmark_impressions]
+select_expression = "COUNT(DISTINCT IF(is_terminal AND result.product_result_type = 'bookmark', event_id, NULL))"
+data_source = "urlbar_events_unnested_results"
+[metrics.bookmark_impressions.statistics.bootstrap_mean]
+
+[metrics.search_suggestion_impressions]
+select_expression = "COUNT(DISTINCT IF(is_terminal AND result.product_result_type = 'default_partner_search_suggestion', event_id, NULL))"
+data_source = "urlbar_events_unnested_results"
+[metrics.search_suggestion_impressions.statistics.bootstrap_mean]
+
+## Clicks
+[metrics.history_clicks]
+select_expression = "COUNTIF(is_terminal and event_action = 'engaged' and product_engaged_result_type = 'history')"
+data_source = "urlbar_events"
+[metrics.history_clicks.statistics.bootstrap_mean]
+
+[metrics.bookmark_clicks]
+select_expression = "COUNTIF(is_terminal and event_action = 'engaged' and product_engaged_result_type = 'bookmark')"
+data_source = "urlbar_events"
+[metrics.bookmark_clicks.statistics.bootstrap_mean]
+
+[metrics.search_suggestion_clicks]
+select_expression = "COUNTIF(is_terminal and event_action = 'engaged' and product_engaged_result_type = 'default_partner_search_suggestion')"
+data_source = "urlbar_events"
+[metrics.search_suggestion_clicks.statistics.bootstrap_mean]
+
+## CTR
+[metrics.history_ctr]
+depends_on = ["history_clicks", "history_impressions"]
+friendly_name = "History CTR"
+description = "Proportion of urlbar sessions where suggestion from the user's browsing history was clicked, out of all urlbar sessions where one was shown (not available on control)"
+
+[metrics.history_ctr.statistics.population_ratio]
+numerator = "history_clicks"
+denominator = "history_impressions"
+
+[metrics.bookmark_ctr]
+depends_on = ["bookmark_clicks", "bookmarks_impressions"]
+friendly_name = "Bookmark CTR"
+description = "Proportion of urlbar sessions where suggestion from the user's bookmarks was clicked, out of all urlbar sessions where one was shown (not available on control)"
+
+[metrics.bookmark_ctr.statistics.population_ratio]
+numerator = "bookmark_clicks"
+denominator = "bookmark_impressions"
+
+[metrics.search_suggestion_ctr]
+depends_on = ["search_suggestion_clicks", "search_suggestion_impressions"]
+friendly_name = "Search Suggestion CTR"
+description = "Proportion of urlbar sessions where suggestion from the default search engine was clicked, out of all urlbar sessions where one was shown (not available on control)"
+
+[metrics.search_suggestion_ctr.statistics.population_ratio]
+numerator = "search_suggestion_clicks"
+denominator = "search_suggestion_impressions"
+
+## Impression Rate
```

Comment:
the impression rate metrics depends on something called "urlbar_impressions" I didn't see it defined anywhere in the original reference file from "fakespot in suggest" https://github.com/mozilla/metric-hub/blob/main/jetstream/fakespot-in-suggest.toml

so i'm assuming that is just a know metric in metric hub and fine to use without something else it depends on.

Reply:
I think it is. I see `urlbar_pressions` in https://github.com/mozilla/metric-hub/blob/main/jetstream/yelp-suggestions-pilot-v3.toml.

Reply:
Strange, I don't see it in `firefox_desktop` definitions. I wonder if it was removed sometime; we'll need to define it either there or in this config.

Reply:
Michael resolved here https://github.com/mozilla/metric-hub/pull/932 - as soon as that lands - this will work.